inlang/source-code/message-lint-rules/messageWithSnakeCaseId/README.md

```diff
@@ -0,0 +1,21 @@
+# What does this rule do?
+
+Checks messages to have a snake case message id.
+
+# How to use
+
+Add the lint rule to your modules array in your `project.inlang.json` file:
+
+```json
```

Comment:
Please explain why snake_case might be beneficial. 

- interop with other environments, like inlang paraglide-js